prismic-model/js/collection-venues.js

```diff
@@ -0,0 +1,109 @@
+import description from './parts/description';
+import timestamp from './parts/timestamp';
+import number from './parts/number';
+const CollectionVenue = {
+  Main: {
+    title: {
+      type: 'Text',
+      config: {
+        label: 'Title'
+      }
+    },
+    description,
+    order: number('Order')
+  },
+  'Regular opening times': {
+    monday: {
+      type: 'Group',
+      fieldset: 'Monday\'s times',
+      config: {
+        repeat: false,
+        fields: {
+          startDateTime: timestamp('Opens'),
+          endDateTime: timestamp('Closes')
+        }
+      }
+    },
+    tuesday: {
```

Comment:
For tidyness you could create this as an object and use it multiple times... nitpick.